ggshield/cmd.py

```diff
@@ -164,7 +164,7 @@ def scan(
 scan = cast(click.Group, json_output_option_decorator(scan))
 
 
-@scan.resultcallback()
+@scan.result_callback()
```

Comment:
As discussed with you, I did it in `f0035d559e` and finally had to revert it in `a607b26d45`. If I remember well, system wide installation using pip can be broken if an older version of `click` is already installed.....

Reply:
This new syntax was added in Click 8.0.0, released in may 2021. What do you think of defining this as the minimum required version?